commit ea6992aa1f6ed514fe450f46befa56d8d2b6a7fb
tree f2043b39f2eb517a25282e5cc727c82132359c6a
parent 9f6c9a8c50bc84ec748fec779ead321ee2b2debc
author Masakazu Mokuno <mokuno@sm.sony.co.jp> 1184919771 +0900
committer Jeff Garzik <jeff@garzik.org> 1185308919 -0400

    ps3: some minor cleanups
    
    - Removed the embarrassing definition which was used in only one
      place.
    - Fixed wrong initialization of dmac_cmd_status.
    
    Signed-off-by: Masakazu Mokuno <mokuno@sm.sony.co.jp>
    Signed-off-by: Jeff Garzik <jeff@garzik.org>

---
 drivers/net/gelic_net.c |    5 +++--
 drivers/net/gelic_net.h |    1 -
 2 files changed, 3 insertions(+), 3 deletions(-)

--- linux-2.6.22.orig/drivers/net/gelic_net.c
+++ linux-2.6.22/drivers/net/gelic_net.c
@@ -292,7 +292,8 @@ static void gelic_net_release_rx_chain(s
 			descr->buf_addr = 0;
 			dev_kfree_skb_any(descr->skb);
 			descr->skb = NULL;
-			descr->dmac_cmd_status = GELIC_NET_DESCR_NOT_IN_USE;
+			gelic_net_set_descr_status(descr,
+						   GELIC_NET_DESCR_NOT_IN_USE);
 		}
 		descr = descr->next;
 	} while (descr != card->rx_chain.head);
@@ -376,7 +377,7 @@ static void gelic_net_release_tx_descr(s
 	descr->skb = NULL;
 
 	/* set descr status */
-	descr->dmac_cmd_status = GELIC_NET_DMAC_CMDSTAT_NOT_IN_USE;
+	gelic_net_set_descr_status(descr, GELIC_NET_DESCR_NOT_IN_USE);
 }
 
 /**
--- linux-2.6.22.orig/drivers/net/gelic_net.h
+++ linux-2.6.22/drivers/net/gelic_net.h
@@ -135,7 +135,6 @@ enum gelic_net_int1_status {
 						      * interrupt status */
 
 #define GELIC_NET_DMAC_CMDSTAT_CHAIN_END  0x00000002 /* RXDCEIS:DMA stopped */
-#define GELIC_NET_DMAC_CMDSTAT_NOT_IN_USE 0xb0000000
 #define GELIC_NET_DESCR_IND_PROC_SHIFT    28
 #define GELIC_NET_DESCR_IND_PROC_MASKO    0x0fffffff
 
